### PR TITLE
Conflict test

### PR DESCRIPTION
**Pull request by iluxa-com**

### CODE REVIEW

**Comment by iluxa-com** [comment]
test comment in review (edit by other user)

File: conflict-text.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1
+2
```
Comment (iluxa-com) [inline conflict-text.txt:2]: second line comment

**Comment by iluxa-c0m** [comment]
i can edit other's user comment

File: conflict-text.txt (hunk 1 above)
Comment (iluxa-com) [reply conflict-text.txt:2]: reply

**Comment by iluxa-com** [comment]
from UP

File: conflict-text.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+1
```
Comment (iluxa-com) [inline conflict-text.txt:1]: 1st line

**Comment by iluxa-c0m** [comment]
PR comment

**Comment by iluxa-c0m** [comment]
review comment 1

File: conflict-text.txt (hunk 2 above)
Comment (iluxa-c0m) [reply conflict-text.txt:1]: reply at 1st line

**Comment by iluxa-com** [comment]
rt

**Comment by iluxa-com** [comment]
who am i in GH

**Pull request merged**